data.sql

```diff
@@ -228,3 +228,365 @@ SET OWNERS_ID =
+ '2020-01-05'
+);
+
+
+INSERT INTO VISITS (
+  AID, 
+  VID,
+  VISITED_DATE
+)
+VALUES 
+(
+ (SELECT ID FROM ANIMALS WHERE NAME = 'Pikachu'),
+ (SELECT ID FROM VETS WHERE NAME = 'Vet Maisy Smith'),
+ '2020-03-08'
+);
+
+
+INSERT INTO VISITS (
+  AID, 
+  VID,
+  VISITED_DATE
+)
+VALUES 
+(
+ (SELECT ID FROM ANIMALS WHERE NAME = 'Pikachu'),
+ (SELECT ID FROM VETS WHERE NAME = 'Vet Maisy Smith'),
+ '2020-05-14'
+);
+
+
+INSERT INTO VISITS (
+  AID, 
+  VID,
+  VISITED_DATE
+)
+VALUES 
+(
+ (SELECT ID FROM ANIMALS WHERE NAME = 'Devimon'),
+ (SELECT ID FROM VETS WHERE NAME = 'Vet Stephanie Mendez'),
+ '2021-05-04'
+);
+
+INSERT INTO VISITS (
+  AID, 
+  VID,
+  VISITED_DATE
+)
+VALUES 
+(
+ (SELECT ID FROM ANIMALS WHERE NAME = 'Charmander'),
+ (SELECT ID FROM VETS WHERE NAME = 'Vet Jack Harkness'),
+ '2021-02-24'
+);
+
+INSERT INTO VISITS (
+  AID, 
+  VID,
+  VISITED_DATE
+)
+VALUES 
+(
+ (SELECT ID FROM ANIMALS WHERE NAME = 'Plantmon'),
+ (SELECT ID FROM VETS WHERE NAME = 'Vet Maisy Smith'),
+ '2019-12-21'
+);
+
+
+
+INSERT INTO VISITS (
+  AID, 
+  VID,
+  VISITED_DATE
+)
+VALUES 
+(
+ (SELECT ID FROM ANIMALS WHERE NAME = 'Plantmon'),
+ (SELECT ID FROM VETS WHERE NAME = 'Vet William Tatcher'),
+ '2020-08-10'
+);
+
+
+INSERT INTO VISITS (
+  AID, 
+  VID,
+  VISITED_DATE
+)
+VALUES 
+(
+ (SELECT ID FROM ANIMALS WHERE NAME = 'Plantmon'),
+ (SELECT ID FROM VETS WHERE NAME = 'Vet Maisy Smith'),
+ '2020-04-07'
+);
+
+
+INSERT INTO VISITS (
+  AID, 
+  VID,
+  VISITED_DATE
+)
+VALUES 
+(
+ (SELECT ID FROM ANIMALS WHERE NAME = 'Angemon'),
+ (SELECT ID FROM VETS WHERE NAME = 'Vet Jack Harkness'),
+ '2020-10-03'
+);
+
+
+INSERT INTO VISITS (
+  AID, 
+  VID,
+  VISITED_DATE
+)
+VALUES 
+(
+ (SELECT ID FROM ANIMALS WHERE NAME = 'Angemon'),
+ (SELECT ID FROM VETS WHERE NAME = 'Vet Jack Harkness'),
+ '2020-11-04'
+);
+
+
+INSERT INTO VISITS (
+  AID, 
+  VID,
+  VISITED_DATE
+)
+VALUES 
+(
+ (SELECT ID FROM ANIMALS WHERE NAME = 'Squirtle'),
+ (SELECT ID FROM VETS WHERE NAME = 'Vet Stephanie Mendez'),
+ '2020-04-07'
+);
+
+
+
+INSERT INTO VISITS (
+  AID, 
+  VID,
+  VISITED_DATE
+)
+VALUES 
+(
+ (SELECT ID FROM ANIMALS WHERE NAME = 'Boarmon'),
+ (SELECT ID FROM VETS WHERE NAME = 'Vet Maisy Smith'),
+ '2019-01-24'
+);
+
+
+INSERT INTO VISITS (
+  AID, 
+  VID,
+  VISITED_DATE
+)
+VALUES 
+(
+ (SELECT ID FROM ANIMALS WHERE NAME = 'Boarmon'),
+ (SELECT ID FROM VETS WHERE NAME = 'Vet Maisy Smith'),
+ '2019-05-15'
+);
+
+
+INSERT INTO VISITS (
+  AID, 
+  VID,
+  VISITED_DATE
+)
+VALUES 
+(
+ (SELECT ID FROM ANIMALS WHERE NAME = 'Boarmon'),
+ (SELECT ID FROM VETS WHERE NAME = 'Vet Maisy Smith'),
+ '2020-02-27'
+);
+
+INSERT INTO VISITS (
+  AID, 
+  VID,
+  VISITED_DATE
+)
+VALUES 
+(
+ (SELECT ID FROM ANIMALS WHERE NAME = 'Boarmon'),
+ (SELECT ID FROM VETS WHERE NAME = 'Vet Maisy Smith'),
+ '2020-08-03'
+);
+
+INSERT INTO VISITS (
+  AID, 
+  VID,
+  VISITED_DATE
+)
+VALUES 
+(
+ (SELECT ID FROM ANIMALS WHERE NAME = 'Boarmon'),
```

Comment:
- [ ] The animal name for `Vet Stephanie Mendez` should be `Blossom` not `Boarmon`